config/crd/bases/infrastructure.cluster.x-k8s.io_tinkerbellclusters.yaml

```diff
@@ -22,26 +22,37 @@ spec:
         description: TinkerbellCluster is the Schema for the tinkerbellclusters API.
         properties:
           apiVersion:
-            description: 'APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
```

Comment:
Regeneration should probably be committed separately.

Reply:
done